src/draco/core/bit_utils.h

```diff
@@ -95,11 +95,19 @@ typename std::make_unsigned<IntTypeT>::type ConvertSignedIntToSymbol(
   if (val >= 0) {
     return static_cast<UnsignedType>(val) << 1;
   }
-  val = -(val + 1);  // Map -1 to 0, -2 to -1, etc..
-  UnsignedType ret = static_cast<UnsignedType>(val);
-  ret <<= 1;
-  ret |= 1;
-  return ret;
+
+  // Prevent warning about applying unary minus to unsigned type
+  if constexpr (std::is_signed_v<IntTypeT>)
```

Comment:
Would this be worth upstreaming so we don't have to apply the fix every upgrade?

Reply:
We could try it; there's tons of int sign related warnings so I'm not sure if they'd accept a fix for one of them or not

Reply:
Ah I see. If this is just a warning and not an actual build error, can this just be removed?

Reply:
The WinUWP build needs it unfortunately, the other platforms were fine with it

Reply:
I think this was only because you had specified warnings in the build that would error on these issues. Now that you've removed them, I don't think you need this anymore.

Reply:
Specifically Zw. If I'm wrong about this, my nitpick would be to change the command from warning to error and then to wrap it in an ifdef block, just for UWP. This will help restrict the different behavior to just that platform and hopefully mitigate any potential unintended consequences.

Reply:
Ok, that makes sense. You're probably right about the warnings, I'll update and rerun the vTest

Reply:
The build still failed even without the flags: https://runtime-rtc.esri.com/me/my-views/view/all/job/vtest/job/dev/job/3rdparty-winuwp/80/parsed_console/

I'll add a `#define` block and bring back the fix